### ISSUE TITLE
Accuracy error: "fastNLP_Bert",

### ISSUE THREAD

**Issue by desertfire** [pytorch/torchdynamo#622]

**Comment by desertfire** [comment]
Repro:
```
python benchmarks/torchbench.py -d cuda --float32 --inductor --no-skip -k   fastNLP_Bert
```
Error:
```
cuda eval  fastNLP_Bert                       Similarity score=-0.34630754590034485
Accuracy failed for key name pred_start
INCORRECT
ERROR
```

**Comment by desertfire** [comment]
Latest error: https://github.com/pytorch/torchdynamo/issues/837

**Comment by yanboliang** [comment]
Fixed by #806
